app/src/test/kotlin/com/wire/android/mapper/UserTypeMapperTest.kt

```diff
@@ -48,7 +48,7 @@ class UserTypeMapperTest {
     @Test
     fun `given internal as a user type correctly map to none as membership`() {
         val result = userTypeMapper.toMembership(UserType.INTERNAL)
-        assertEquals(Membership.None, result)
+        assertEquals(Membership.Standard, result)
```

Comment:
This test was not running, since the annotations were junit4, being skipped.
Now is running with this PR and this check was broken.